store/tooling/tooling.go

```diff
@@ -357,7 +363,8 @@ func (tsto *ToolingStore) DownloadMany(toDownload []SnapToDownload, curSnaps []*
 	return downloadedSnaps, nil
 }
 
-// AssertionFetcher creates an asserts.Fetcher for assertions using dlOpts for authorization, the fetcher will add assertions in the given database and after that also call save for each of them.
+// AssertionFetcher creates an asserts.Fetcher for assertions using dlOpts for authorization,
```

Comment:
As we are touching this, I'm not sure what dlOpts is referring to here, seems 'using ... for authorization' can go away?